app/controllers/api/physical_storages_controller.rb

```diff
@@ -4,12 +4,11 @@ def refresh_resource(type, id, _data = nil)
       enqueue_ems_action(type, id, "Refreshing", :method_name => :refresh_ems)
     end
 
-    def create_resource(_type, _id = nil, data = {})
-      ext_management_system = ExtManagementSystem.find(data['ems_id'])
-      task_id = PhysicalStorage.create_physical_storage_queue(session[:userid], ext_management_system, data)
-      action_result(true, "Creating Physical Storage #{data['name']} for Provider: #{ext_management_system.name}", :task_id => task_id)
-    rescue => err
-      action_result(false, err.to_s)
+    def create_resource(type, _id = nil, data = {})
+      # NOTE: no supports AND not ems specific physical storage
```

Comment:
@kbrock can we keep a running list of things like this that are inconsistent?  That way I can go through and fix them

Reply:
`NOTE:` can be that running list (or perhaps `TODO:` might be better).  Formatting the comments in this consistent way allow tools to extract them as TODO lists, such as with https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree

Reply:
@Fryguy Did you ask for me to use `TODO`? Or was there something more to this request?

Reply:
I was asking @agrare if `TODO:` is sufficient as that running list of things or not, considering there are tools that can be used to find them.

Reply:
Okay a TODO is fine, was thinking we could put them in an issue with a checklist but I can create that from these TODOs

Reply:
`ag "create_ems[^:]*$"`